### DISCUSSION TITLE
原神代肝

### DISCUSSION THREAD

**Discussion by komhshan** [share121/inter-knot#5628] [公告]
![Collage_20241024_150855](https://github.com/user-attachments/assets/dafcb316-3180-47ff-a39b-e5feaf5ba4cf)
希望大家能来光顾我的生意

**Comment by yingguan-Cloud** [comment]
看来不止我一个代肝生意不太好（主要补探索的单子费时间）

**Reply by komhshan** [reply]
没办法，我想接补探索度的杆我都接不到
